src/libraries/System.Private.CoreLib/src/System/ArgumentNullException.cs

```diff
@@ -64,6 +64,18 @@ public static void ThrowIfNull([NotNull] object? argument, [CallerArgumentExpres
             }
         }
 
+        /// <summary>Throws an <see cref="ArgumentNullException"/> if <paramref name="argument"/> is null.</summary>
+        /// <param name="argument">The pointer argument to validate as non-null.</param>
+        /// <param name="paramName">The name of the parameter with which <paramref name="argument"/> corresponds.</param>
+        [CLSCompliant(false)]
+        public static unsafe void ThrowIfNull([NotNull] void* argument, [CallerArgumentExpression("argument")] string? paramName = null)
+        {
+            if (argument is null)
```

Comment:
Just calling it out, because I always forget, `x is null` does work for pointers today.